packages/s3-cache/src/index.ts

```diff
@@ -58,14 +59,15 @@ export class S3Cache implements CacheStrategy {
   }
 
   async set(pageKey: string, cacheKey: string, data: CacheEntry): Promise<void> {
-    const baseInput = {
+    const baseInput: PutObjectCommandInput = {
```

Comment:
```suggestion
    const baseInput: Partial<PutObjectCommandInput> = {
```

Reply:
nit: since we are going to have only some predefined values here, but not the whole command object

Reply:
It breaks typing mapping for the next operations. Can we leave it?